scripts/generate-mjml-react.ts

```diff
@@ -52,30 +54,14 @@ const MJML_COMPONENT_NAMES = MJML_COMPONENTS_TO_GENERATE.map(
   (component) => component.componentName
 );
 
-const ATTRIBUTES_TO_USE_CSSProperties_WITH = new Set([
-  "color",
-  "textAlign",
-  "verticalAlign",
-  "textDecoration",
-  "textTransform",
-
-  "border",
-  "borderRadius",
-  "borderColor",
-  "borderStyle",
-
-  "backgroundColor",
-  "backgroundPosition",
-  "backgroundRepeat",
-  "backgroundSize",
-]);
-
 const TYPE_OVERRIDE: { [componentName: string]: { [prop: string]: string } } = {
   mjml: { owa: "string", lang: "string" },
   "mj-style": { inline: "boolean" },
   "mj-class": { name: "string" },
   "mj-table": { cellspacing: "string", cellpadding: "string" },
   "mj-selector": { path: "string" },
+  "mj-section": { fullWidth: "boolean" },
+  "mj-wrapper": { fullWidth: "boolean" },
```

Comment:
I pulled these overrides out of the more generic `getPropTypeFromMjmlAttributeType`